jobbergate_cli/main.py

```diff
@@ -199,9 +199,15 @@ def init_token(username, password):
         JOBBERGATE_API_OBTAIN_TOKEN_ENDPOINT,
         data={"email": username, "password": password},
     )
+    if not resp.ok:
+        logger.error(f"Failed to retrieve a token, got response: {resp.text}")
+        resp.raise_for_status()
+
     data = resp.json()
-    ret = data.get("token")
-    JOBBERGATE_API_JWT_PATH.write_text(ret)
```

Comment:
This was causing the issue. There was no token on the response, leading to `ret=None`, and a `TypeError` raised from `write_text`.